### PR TITLE
whatshap stats - Enable comma-separated chromosomes as argument

### LINKED ISSUE #207: Allow `--chromosome` to accept a comma-delimited list of chromosomes.
**[Original report](https://bitbucket.org/whatshap/whatshap/issue/207) by William Rowell (Bitbucket: [williamrowell](https://bitbucket.org/williamrowell), GitHub: [williamrowell](https://github.com/williamrowell)).**

----------------------------------------

I would like to calculate phase block size only for autosomes in a male, and while I can use tabix to remove the sex chromosomes, it would be convenient to pass a list of autosomes to whatshap stats.  Would it be possible to allow the `--chromosome` argument for `whatshap phase` and/or `whatshap stats` to accept a comma delimited list as an argument?

### PR DESCRIPTION
**Pull request by pontushojer**
fix #207

### CODE REVIEW

File: whatshap/cli/stats.py [hunk 1]
```diff
@@ -27,8 +27,8 @@ def add_arguments(parser):
         "and ignore all other variants.")
     add("--block-list", metavar="FILE", help="Write list of all blocks to FILE (one block per line)")
     add("--chromosome", dest="chromosomes", metavar="CHROMOSOME", default=[], action="append",
-        help="Name of chromosome to process. If not given, all chromosomes in the "
-        "input VCF are considered. Can be used multiple times")
+        help="Name of chromosome(s) to process. If not given, all chromosomes in the "
+        "input VCF are considered. Accepts multiple inputs or a comma-separated list.")
```
Comment (marcelm) [inline whatshap/cli/stats.py:31]: "Accepts multiple inpus" could also mean that you can write it like `--chromosome chr1 chr2 chr3` (argparse has support for this if I remember correctly). I think explicitly stating that the option can be used multiple times is less ambiguous.

File: whatshap/cli/stats.py [hunk 2]
```diff
@@ -319,6 +319,12 @@ def get_detailed_stats(self, chr_lengths: Optional[Dict[str, int]] = None) -> De
             )
 
 
+def unpack_chromosomes(chromosomes: List[str]) -> List[str]:
+    """Unpack list chromosomes by splitting comma-separete entries."""
```
Comment (marcelm) [inline whatshap/cli/stats.py:323]:
```suggestion
    """Unpack list of chromosomes by splitting comma-separated entries."""
```

**Review by marcelm — COMMENTED** [review]
There’s a merge conflict in the changelog because I just merged another PR, sorry about that.

**Comment by pontushojer** [comment]
> There’s a merge conflict in the changelog because I just merged another PR, sorry about that.

No worry! Fixed the comments.

**Comment by marcelm** [comment]
Perfect, thanks!

**Pull request merged**